packages/grafana-data/src/types/app.ts

```diff
@@ -126,6 +126,10 @@ export class AppPlugin<T extends KeyValue = KeyValue> extends GrafanaPlugin<AppP
  * @internal
  */
 export enum FeatureState {
+  /** @deprecated in favor of experimental */
   alpha = 'alpha',
```

Comment:
I could be wrong here but isn't this related to the [plugins.alpha](https://github.com/grafana/grafana/blob/main/conf/defaults.ini#L1592) config setting? 🤔

Reply:
That seems to make sense. In that case, maybe it should show up as `Experimental` in the UI instead? Although it will lead to confusion in the config, but be more consistent with the Grafana release life cycle.

Possibly that a mapping between `Alpha -> Experimental` and `Beta -> Preview`.

Reply:
When I looked how this is used, I haven't seen anything related to plugin/dependency on this setting:  https://github.com/search?q=org%3Agrafana+%22FeatureState.%22&type=code

Reply:
That's true, it might just use a `Badge`